server/tserver/src/main/java/org/apache/accumulo/tserver/InMemoryMap.java

```diff
@@ -683,13 +683,6 @@ public void setInterruptFlag(AtomicBoolean flag) {
     private SourceSwitchingIterator ssi;
     private MemoryDataSource mds;
 
-    @Override
-    protected SortedKeyValueIterator<Key,Value> getSource() {
-      if (closed.get())
-        throw new IllegalStateException("Memory iterator is closed");
-      return super.getSource();
-    }
-
```

Comment:
It could just be a margin of error or the JMH test I wrote: https://github.com/milleruntime/jmh-test/blob/6cd94569bbecda42181b6b7d7101e0770ef00d7f/src/main/java/org/sample/InMemoryMapBenchmark.java